plugins/http_plugin/http_plugin.cpp

```diff
@@ -776,7 +776,16 @@ class http_plugin_impl : public std::enable_shared_from_this<http_plugin_impl> {
          EOS_ASSERT( my->thread_pool_size > 0, chain::plugin_config_exception,
                      "http-threads ${num} must be greater than 0", ("num", my->thread_pool_size));
 
-         my->max_bytes_in_flight = options.at( "http-max-bytes-in-flight-mb" ).as<uint32_t>() * 1024 * 1024;
+         // http-max-bytes-in-flight-mb is allowed to be -1,
+         // that's why int64_t is used
```

Comment:
Would not mention this except I have the other comment. This could be on the same line or even just remove the second line of the comment.